arithmetization/src/test/java/net/consensys/linea/ZkCounterTest.java

```diff
@@ -376,4 +385,83 @@ private static Stream<Arguments> modexpInput() {
     }
     return arguments.stream();
   }
+
+  private static Stream<Arguments> blsAndKzgInput() {
+    final List<Arguments> arguments = new ArrayList<>();
+    for (Address address : BLS_PRECOMPILES) {
+      arguments.add(Arguments.of(address));
+    }
+    arguments.add(Arguments.of(KZG_POINT_EVAL));
+    return arguments.stream();
+  }
+
+  @ParameterizedTest
+  @MethodSource("blsAndKzgInput")
+  void blsAndKzg(Address prc) {
+    // sender account
+    final KeyPair senderKeyPair = new SECP256K1().generateKeyPair();
+    final Address senderAddress =
+        Address.extract(Hash.hash(senderKeyPair.getPublicKey().getEncodedBytes()));
+    final ToyAccount senderAccount =
+        ToyAccount.builder().balance(Wei.fromEth(123)).nonce(12).address(senderAddress).build();
+
+    // receiver account: calls PRC
+    final ToyAccount callPRC =
+        ToyAccount.builder()
+            .balance(Wei.fromEth(1))
+            .address(Address.wrap(Bytes.repeat((byte) 1, Address.SIZE)))
+            .code(
+                BytecodeCompiler.newProgram(testInfo)
+                    // populate memory with some data
+                    .push(56) // value
+                    .push(3) //  offset
+                    .op(OpCode.MSTORE8)
+                    // call the precompile
+                    .push(1) // return size
+                    .push(0) // return offset
+                    .push(0) // cds
```

Comment:
Note: none of the Cancun / Prague precompiles accept empty call data. I don't know if Besu will register a precompile call if there is a call data size violation, and thus not sure if the test triggers anything.